FlaskSite/__init__.py

```diff
@@ -16,24 +18,9 @@ def createApp():
     # Enable CORS (Adjust as needed)
     CORS(app, resources={r"/api/*": {"origins": "http://localhost:3000"}}) # restrict it to specific origins React deafult clinet 
 
-    configPath = os.path.join(os.path.abspath(os.path.dirname(__file__)), 'config.py')
-    # Load configurations
-    if not fileExists(configPath):
-        print("config is not found")
-        content = (
-        "import os\n"
-        "# Default configuration settings\n"
-        "DEBUG = False\n"
-        "SECRET_KEY = 'Make It Hard'\n"
-        "BASE_DIR = os.path.abspath(os.path.dirname(__file__))\n"
-        "SQLALCHEMY_DATABASE_URI = 'sqlite:///' + os.path.join(BASE_DIR, 'instance', 'site.db')\n"
-        "IMG_FOLDER = os.path.join(BASE_DIR, 'instance', 'uploads')\n"
-            )
-        
-        writeToFile(configPath, content)
-        print(f"config.py created with default settings.")
-
-    app.config.from_pyfile(configPath)
+    # Load configurations based on the environment
+    env = os.environ.get('FLASK_ENV', 'default')  # Default to 'default' if FLASK_ENV is not set Hint: FLASK_ENV=development && flask run
```

Comment:
move this to a method get_enviornment()

Reply:
on a second look, what about create a method that return the config for this environment, after the change it should look like that
app.config.from_object(get_config())
and let get_config() handle how to get the env